data/json/mutations/mutations.json

```diff
@@ -3598,6 +3598,28 @@
       "strength_damage": { "damage_type": "cut", "amount": 4 }
     }
   },
+  {
+    "type": "mutation",
+    "id": "TOE_CLAWS",
+    "name": { "str": "Clawed Feet" },
+    "copy-from": "RAP_TALONS",
```

Comment:
This will function exactly the same as Raptor Talons and use the same attack even though you are talking about multiple claws compared to a single talon?  At the very least the attack text should change.

Reply:
I did update that but somehow it didn't make it into the PR.